xbmc/video/dialogs/GUIDialogVideoSettings.cpp

```diff
@@ -259,8 +259,6 @@ void CGUIDialogVideoSettings::InitializeSettings()
   entries.push_back(std::make_pair(16318, VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL_HALF));
   entries.push_back(std::make_pair(16317, VS_INTERLACEMETHOD_VDPAU_TEMPORAL_HALF));
   entries.push_back(std::make_pair(16314, VS_INTERLACEMETHOD_VDPAU_INVERSE_TELECINE));
-  entries.push_back(std::make_pair(16320, VS_INTERLACEMETHOD_DXVA_BOB));
```

Comment:
is that so? I am not sure what a spinner does when selecting a value not added. if they are robust enough, then fine.